Cargo.toml

```diff
@@ -1,28 +1,24 @@
 [workspace]
 resolver = "2"
 members = [
-    "gateway/crates/*",
-    "cli/crates/*",
-    "engine/crates/*",
-    "engine/crates/engine-v2/auth",
-    "engine/crates/engine-v2/config",
-    "engine/crates/engine-v2/axum",
-    "engine/crates/engine-v2/codegen",
-    "engine/crates/engine-v2/walker",
-    "engine/crates/engine-v2/schema",
-    "engine/crates/engine-v2/query-solver",
-    "engine/crates/engine-v2/id-derives",
-    "engine/crates/engine-v2/id-newtypes",
-    "engine/crates/engine-v2/parser",
-    "engine/crates/engine-v2/value",
-    "graph-ref",
-    "graphql-lint",
-    "gqlint",
-    "workspace-hack",
+    "cli",
+    "gateway",
+    "crates/*",
```

Comment:
can't we do "crates/**/*" or "crates/engine-v2/*" here?

Reply:
No, because engine-v2 isn't a repository that contains only crates (just `src/` is enough to break that), that wouldn't work.